### ISSUE TITLE
pretrained_models/bert-base-uncased model cannot found error

### ISSUE THREAD

**Issue by hjjeong** [xnliang98/uke_ccrank#1]
The predefined model 'pretrained_models/bert-base-uncased' in get_embedding.py raises RepositoryNotFoundError.

![image](https://user-images.githubusercontent.com/8444098/155090561-52fd66f0-bfd6-49b2-8c7a-c4ee8b9e9844.png)

So, I modify the model name from 'pretrained_models/bert-base-uncased' to 'bert-base-uncased' to use the bert model of huggingface transformers.
But, it also raises AttributeError in o2.squeeze().detach().numpy() function(line 58).
![image](https://user-images.githubusercontent.com/8444098/155089757-2c0d858d-b4f5-48f0-baa4-78808ce1f23e.png)

Could you tell me which the BERT model you used?
or Could you tell me why I am getting this error?

**Comment by jamesjohnson1025** [comment]
Why this is closed ?. Me too facing the same issue. Is there a solution to fix the second error ?.

**Comment by xnliang98** [comment]
> 
If you can run this code with the Internet, you can change the 'pretrained_models/bert-base-uncased' to 'bert-base-uncased', then the code will download the model from Huggingface Hub.

Or you can download it from https://huggingface.co/bert-base-uncased manually and then place it in dir 'pretrained_models/bert-base-uncased'.
